testing/scenario_app/lib/README.md

```diff
@@ -0,0 +1,16 @@
+# `package:scenario_app`
+
+This package simulates a Flutter app that uses the engine (`dart:ui`) only. It
+is used to test the engine in isolation from the rest of the Flutter framework
+and tooling.
+
+## Adding a New Scenario
```

Comment:
This should maybe be at the README.md one level higher.

Reply:
Yeah fair, once I wrote this out it doesn't seem impactful enough.

Reply:
Done